ocean_provider/routes/consume.py

```diff
@@ -145,6 +145,7 @@ def initialize():
         did = data.get("documentId")
         consumer_address = data.get("consumerAddress")
         compute_env = data.get("computeEnv")
+        valid_until = data.get("validUntil", 0)
```

Comment:
please add this to provider_requests.py in validation as "sometimes, number". See other examples for exact config.